### ISSUE TITLE
Consider debouncing

### ISSUE THREAD

**Issue by kurtextrem** [williamngan/roll#3]
http://www.html5rocks.com/en/tutorials/speed/animations/

**Comment by williamngan** [comment]
Yep, we can consider adding it as an option. Though it may be simple enough for users to use `_.debounce` (underscore/lodash) or implement their own as needed. Thoughts?

**Comment by kurtextrem** [comment]
Yeah right. But most people don't know about that issue. Maybe adding it to the readme would help?
